dapps/onboarding.vue/src/components/sign-up/sign-up.vue

```diff
@@ -18,7 +18,7 @@
 */
 
 <template>
-  <div class="row h-100" style="min-height:700px">
```

Comment:
I think min-height was for very small displays, would it still work on a 500px super old smartphone?

Reply:
fixes the issue without min-height specification. Persisting it to 700px will lead to the same problem within the mnemonic formular.